documentation/md/collection/biconnected.md

```diff
@@ -0,0 +1,20 @@
+## Details
+
+Note that this function searches for biconnected components only within the subset of the graph in the calling collection.
+
+This function returns an object of the following form:
+
+```js
+{
+  cutVertices, /* Collection of nodes identified as cut vertices */
+  components /* An array, whose members are collections of edges corresponding to each biconnected component */
```

Comment:
If they're components, they should probably include the nodes connected to the edges.  In most cases, I expect the consumer would be interested in the nodes in each component rather than the edges.  Having a collection of both allows the consumer some simplicity and flexibility.

Maybe `kargerStein()` should also be updated to return `components` with both edges and nodes in each component.  That would make the functions more consistent.

Reply:
This seems reasonable to me. Supposing that `kargerStein()` was modified in this manner, if the consumer wanted `components` to be more faithful to the mathematical definition of a cut they could always filter either collection to contain only nodes.

If you'd like, I'd be happy to go ahead and make that modification either here or in a separate PR, whichever you think would be more appropriate.

Reply:
Might as well just keep things simple and reuse this PR.  Thanks